glfw31-gl41core-cube/cube.go

```diff
@@ -40,7 +39,8 @@ func main() {
 	glfw.WindowHint(glfw.ContextVersionMinor, 1)
 	glfw.WindowHint(glfw.OpenGLProfile, glfw.OpenGLCoreProfile)
 	glfw.WindowHint(glfw.OpenGLForwardCompatible, glfw.True)
-	window, err := glfw.CreateWindow(WindowWidth, WindowHeight, "Cube", nil, nil)
+	window, err := glfw.CreateWindow(windowWidth, windowHeight, "Cube", nil, nil)
+
 	if err != nil {
```

Comment:
I think it's unhelpful to add a blank line between something that returns `err` and checking the value of `err`. None of the code in Go standard library does it. Search for "if err" on https://golang.org/doc/effective_go.html, as an example.